doc/email-notificaitons.md

```diff
@@ -0,0 +1,45 @@
+# Email notifications
+The application sends various notifications by email, this document outlines how
+this works from a technical standpoint.
+
+## Account emails
+Most emails that relate to user accounts are sent by Auth0.
+
+All others are sent by the applicaiton via GOV.UK Notify.
+
+## Mail Notify Gem
+We use this gem to interact with the GOV.UK Notify service.
+
+[https://github.com/dxw/mail-notify](https://github.com/dxw/mail-notify)
+
+## GOV.UK notify service
+[https://www.notifications.service.gov.uk/services/15322373-3ee0-4c9c-aef7-be3e967f2f3e](https://www.notifications.service.gov.uk/services/15322373-3ee0-4c9c-aef7-be3e967f2f3e)
+
+BEIS owned so access can be granted by them.
+
+You will need the API key in the NOTIFY_KEY environment variable. The key for
+each environment can be found here https://www.notifications.service.gov.uk/services/15322373-3ee0-4c9c-aef7-be3e967f2f3e/api/keys
+
+## Templates
```

Comment:
It might be worth mentioning how we use templates here (i.e. we send the entire template to Notify, rather than having a template with variables stored in Notify)

Reply:
I didn't know this? I thought we sent the entire body but there is still a templat in Notify?

Reply:
Oh there is, but there's just one variable in it called `body` or something similar

Reply:
https://github.com/dxw/mail-notify#with-a-view